### PR TITLE
Revisions and overall document flow

### PR DESCRIPTION
**Pull request by dhimmel**
WIP! will ping when review is ready. At this point, I'd suggest holding off on most review since the changes are highly unstable.

- [x] Closes https://github.com/greenelab/meta-review/issues/22

### CODE REVIEW

**Comment by agitter** [comment]
I asked a colleague to read a meta review draft to make sure it is accessible to a general audience, but I'll wait until this is finished and merged.

**Comment by agitter** [comment]
@dhimmel is this still a work in progress?  Should I provide comments on what you have here?

**Comment by dhimmel** [comment]
Okay ready for review... not because I completed what I previously wanted to do, but because I cannot remember exactly what I was planning. This is a good lessen to not let PRs linger, or at least note what's left to do.

Let me know if the translocation makes reviewing the diff too difficult.

Let's review what's here and I can continue working on additional changes as they come to mind.

File: content/02.main-text.md
Comment (cgreene) [inline content/02.main-text.md]: `reviews` -> `review` or remove ~articles~

File: content/02.main-text.md
Comment (cgreene) [inline content/02.main-text.md]: Maybe remove `help` from the final bit of this?

File: content/02.main-text.md
Comment (cgreene) [inline content/02.main-text.md]: Expertise

File: content/02.main-text.md
Comment (cgreene) [inline content/02.main-text.md]: What about:
> Writing review articles in a public forum allows review authors to engage with the original researchers to clarify their methods and results and present them accurately, as exemplified [here](https://github.com/greenelab/deep-review/issues/213)

File: content/02.main-text.md
Comment (cgreene) [inline content/02.main-text.md:45]: is `structured` the right word here? It feels more like `aided` but maybe I'm more a prisoner to my tools than I think. :)

File: content/02.main-text.md
Comment (cgreene) [inline content/02.main-text.md]: `limits` -> `limited`

File: content/02.main-text.md
Comment (cgreene) [inline content/02.main-text.md:33]: I like this coming earlier, but I wonder if our dive into the technical side is too deep too fast. I don't think it should hold up this PR but we may want to consider how accessible this is to a general audience as we move forward.

**Review by cgreene — COMMENTED** [review]
some initial thoughts

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: Changing `"The Deep Review"` to `the "Deep Review"` better fits what appears below, and is my preference as well.

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md:7]: With these lines being moved or replaced, we quickly jump to review articles being written in private.  Does that make too strong an assumption that the reader knows what we mean by review article, and should we add back a version of `Reviews presenting the state of the art in a scientific field`?

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: I like that we more explicitly refer to the Deep Review now, but I want to make sure that these early paragraphs don't make it sound like the article is about the history of the Deep Review.  That isn't of general interest to a broad audience.  We want to keep the focus on the `problems or shortcomings in the world of science`, specifically writing closed reviews, and `suggest solutions to these problems` (taken from the eLife feature article guidelines).

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md:113]: These two lines will not be understandable to a general audience.

**Review by agitter — CHANGES_REQUESTED** [review]

File: content/02.main-text.md
Comment (cgreene) [reply content/02.main-text.md]: I think we should defer this consideration to a subsequent PR. I've created an issue so that we don't lose this in #31

Comment (agitter) [reply content/02.main-text.md]: Agreed, let's merge this quickly after making a few minor edits for phrasing.

File: content/02.main-text.md
Comment (dhimmel) [reply content/02.main-text.md:113]: What should we do when we encounter a technically important point that would be prohibitive to explain to a layperson?

Comment (cgreene) [reply content/02.main-text.md:113]: Since I know a lot less about blockchain/bitcoin than @dhimmel, I am going to create a new issue and I will try to rewrite this paragraph such that it is easy for me to understand. That should be a reasonable first step towards a general audience. 👍

File: content/02.main-text.md
Comment (dhimmel) [reply content/02.main-text.md:33]: Yeah, I moved the text around since before there was poor flow from between the different aspects of the paper (collaborative reviews, deep reviews, manubot). There's definitely still work to do in this regard after this PR is merged.

**Review by agitter — APPROVED** [review]
The edits look good to me.  I'm ready to merge and let @cgreene take a pass at #31 or #32.

**Pull request merged**
